### PR TITLE
Feature/modern theme

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/components/AccountDetails/index.tsx [hunk 1]
```diff
@@ -299,113 +271,78 @@ export default function AccountDetails({
...
+              <Wallet size={16} />
+              <Text fontSize="14px" marginLeft="4px">
+                <Trans>Analyze Wallet</Trans> ↗
+              </Text>
+            </Flex>
+          </ExternalLink>
+        </Flex>
+
+        <Flex justifyContent="space-between" marginTop="24px" paddingX="20px" sx={{ gap: '1rem' }}>
+          {/* <ButtonOutlined> */}
```
[the remark this answers is not in the document]
Comment (viet-nv) [reply src/components/AccountDetails/index.tsx:330]: no, we will implement it later

File: src/components/FeeSelector/hook.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        )
+
+        const totalPostions = feeArray.reduce((total, cur) => total + cur.activePositions, 0)
+
+        if (!totalPostions) return
+        setFeeTierDistribution(
+          Object.keys(FeeAmount).reduce((acc, cur) => {
+            const temp = feeArray.find(item => item.feeTier === cur)
+            return {
+              ...acc,
```
Comment (hungdoansy) [inline src/components/FeeSelector/hook.ts:62]: No need to spread `acc` here

Comment (viet-nv) [reply src/components/FeeSelector/hook.ts:62]: what?

**Review by hungdoansy — APPROVED** [review]
LGTM

**Pull request merged**